hswidget/cof_widget.go

```diff
@@ -0,0 +1,660 @@
+package hswidget
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ianling/giu"
+	"github.com/ianling/imgui-go"
+
+	"github.com/OpenDiablo2/OpenDiablo2/d2common/d2enum"
+	"github.com/OpenDiablo2/OpenDiablo2/d2common/d2fileformats/d2cof"
+
+	"github.com/OpenDiablo2/HellSpawner/hscommon/hsenum"
+)
+
+const (
+	indicatorSize                        = 64
+	upDownArrowW, upDownArrowH           = 15, 15
+	leftRightArrowW, leftRightArrowH     = 15, 15
```

Comment:
we can joint into one line
```golang
arrowButtonW, arrowButtonH = 15, 15
```
(generally it is constant for all arrow buttons, so maybe we can put it somewhere in hsenum

Reply:
or perhaps just one constant?

Reply:
also should be ok

Reply:
resolved in 744fe55